lib/create-webpack-config-base.js

```diff
@@ -29,19 +29,49 @@ function createWebpackConfigBase(batfishConfig) {
     const prefixUrlPath = result[1];
 
     const babelPresets = ['es2015', 'react'];
+
+    // Plugins
+    const jsxtremeMarkdownOptions = _.clone(
+      batfishConfig.jsxtremeMarkdownOptions
+    );
+    if (!jsxtremeMarkdownOptions.modules) jsxtremeMarkdownOptions.modules = [];
+    jsxtremeMarkdownOptions.modules.push(
+      `const prefixUrl = require('batfish/prefix-url')`
+    );
+    jsxtremeMarkdownOptions.modules.push(
+      `const routeTo = require('batfish/route-to')`
+    );
+
+    let postcssPlugins = [
+      autoprefixer({ browsers: batfishConfig.autoprefixerBrowsers })
+    ];
+    if (batfishConfig.postcssPlugins) {
+      postcssPlugins = postcssPlugins.concat(batfishConfig.postcssPlugins);
+    }
+
     const babelPlugins = [
       'transform-class-properties',
       'syntax-dynamic-import',
       [
         '@mapbox/babel-plugin-transform-jsxtreme-markdown',
-        { packageName: 'batfish/md' }
+        {
+          packageName: 'batfish/md',
+          remarkPlugins: jsxtremeMarkdownOptions.remarkPlugins || null,
+          rehypePlugins: jsxtremeMarkdownOptions.rehypePlugins || null
```

Comment:
That makes way more sense. I updated this in https://github.com/mapbox/batfish/pull/75/commits/12d4c23af2135ae48a5e7ada34c9f7d5b288d6a3.